chapters/annotations.tex

```diff
@@ -49,10 +49,12 @@ \section{Annotations for Documentation}\label{annotations-for-documentation}
 giving a textual description, the \lstinline!revisions! annotation giving a list
 of revisions and other annotations defined by a tool.
 How the tool interprets the information in \lstinline!Documentation! is
-unspecified. Within a string of the \lstinline!Documentation! annotation, the
-tags \lstinline!<HTML>! and \lstinline!</HTML>! or
-\lstinline!<html>! and \lstinline!</html>! define
-optionally begin and end of content that is HTML encoded. For external
+unspecified. If the string starts with the tag
+\lstinline!</html>! or \lstinline!<HTML>! the entire contents is HTML encoded
```

Comment:
This should say `<html>`, not `</html>`?

To me "are" sounds better than "is" when talking about "contents" (as opposed to "content") in this and following text. I'm no English expert though.

Reply:
1. Yes.
2. I changed to "string" to avoid that discussion. As far as I understand "The contents is" is sometimes correct.